### DISCUSSION TITLE
combined battery power - sensor

### DISCUSSION THREAD

**Discussion by JanKraslice** [kellerza/sunsynk#370] [Q&A]
Hello, I didn't find it in the documentation or in the sensors, but maybe I missed it.

Is there a sensor combined battery power? 
Three-phase inverter HV Deye / sunsysnk 
Two separate batteries each tower connected by its own BMS 1 and BMS 2

Do we have separate sensors like Battery 1 power and Battery 2 power do we have a combined sensor?

**Comment by Waz-Cpt** [comment]
Hello There,

From what I remember you can add battery 1 and battery 2 data as sensor options.

As you can see below, battery 1 and 2 have power sensors defined.

`SENSORS += (
    TempSensor(586, "Battery 1 temperature", CELSIUS, 0.1),
    Sensor(587, "Battery 1 voltage", VOLT, 0.1),
    Sensor(588, "Battery 1 SOC", "%"),
    Sensor(590, "Battery 1 power", WATT, -10),
    Sensor(591, "Battery 1 current", AMPS, -0.01),
    Sensor(589, "Battery 2 SOC", "%"),
    Sensor(593, "Battery 2 voltage", VOLT, 0.1),
    Sensor(594, "Battery 2 current", AMPS, -0.01),
    Sensor(595, "Battery 2 power", WATT, -10),
    TempSensor(596, "Battery 2 temperature", CELSIUS, 0.1),
)
`

Remember that with the HV setup, you would need to specify battery 1 and battery 2 and not just battery.

**Reply by JanKraslice** [reply]
I have two batteries (see picture) 
We have a sensor Day Battery Charge and Day Battery discharge

As it is possible with DEYE high voltage, to have two separate batteries it is possible to have a different value of energy delivered for each battery on a given day see picture.

Is it possible to sensor ?
Day_Battery_1_Charge / Day_Battery_1_discharge
Day_Battery_2_Charge / Day_Battery_2_discharge
![Two_Batter3](https://github.com/user-attachments/assets/04d65585-9a72-47ed-8662-4e653e57dfa8)

**Reply by kellerza** [reply]
Recently did some updated to these (use the edge version)

See the names here - https://github.com/kellerza/sunsynk/commit/e6a5c0b6bb343ec01d69be11eac33a62fa75f117
